packages/web/src/components/gcds-input/gcds-input.tsx

```diff
@@ -188,18 +188,23 @@ export class GcdsInput {
   };
 
   /**
-   * Update value based on user input.
+   * Emitted when the input has received input.
```

Comment:
While we're here updating documentation...
```suggestion
   * Emitted when the element has received input.
```
Open to others, just thought "input has received input" sounds a bit confusing